contentpool_client.services.yml

```diff
@@ -1,8 +1,12 @@
 services:
+  contentpool_client.logger:
+    parent: logger.channel_base
+    arguments: [ 'contentpool_client' ]
```

Comment:
does not match pre-existing type 'contentpool client'

Reply:
@fago agreed. I'd still peffer to change logging on other places to match name with underline since this is common drupal practice